### PR TITLE
FairItem touch area fix

### PR DESCRIPTION
**Pull request by junlee91**

### CODE REVIEW

File: src/screens/Profile/container.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import React, { Fragment } from 'react';
```
Comment (sonamoo) [inline src/screens/Profile/container.js]: Was Fragment unused before?

File: src/components/FairItem/presenter.js [hunk 2]
```diff
@@ -8,21 +8,25 @@ const FairItem = (props) => {
   fair = { company_url: props.company_url || 'uic.edu' };
 
   return (
-    <TouchableOpacity onPressOut={() => { props.navigateTo(props.fair.id); }}>
+    <>
```
Comment (sonamoo) [inline src/components/FairItem/presenter.js]: what does <> do?

File: src/screens/Profile/container.js (hunk 1 above)
Comment (junlee91) [reply src/screens/Profile/container.js]: Yup . 
And there is a better way of writing `Fragment`

```
<>
   <Component />
</>
```

File: src/components/FairItem/presenter.js (hunk 2 above)
Comment (junlee91) [reply src/components/FairItem/presenter.js]: https://reactjs.org/docs/fragments.html#short-syntax .

**Pull request merged**